discovery-provider/src/queries/get_full_save_tracks.py

```diff
@@ -0,0 +1,58 @@
+from src.models import Track, Save, SaveType
+from src.utils import helpers
+from src.utils.db_session import get_db_read_replica
+from src.queries import response_name_constants
+from src.queries.query_helpers import add_query_pagination, populate_track_metadata, \
+    get_users_by_id, get_users_ids
+
+def get_full_save_tracks(args):
```

Comment:
I think we should just call this get_save_tracks or something like that. The "full" vs. "not full" thing is not really part of the query layer so far